vm/src/vm.rs

```diff
@@ -58,18 +58,22 @@ impl VirtualMachine {
         let ctx = PyContext::new();
 
         // Hard-core modules:
-        let builtins = builtins::make_module(&ctx);
```

Comment:
This is wrong, make_module actually add's builtins to the scope right? New module simply creates an empty module. This might be the source of the build errors.

Reply:
Ignore my comment, I now see the code below :).